bundles/org.openhab.binding.homeconnect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -233,6 +240,7 @@
 		</supported-bridge-type-refs>
 		<label>Coffee Machine</label>
 		<description>Home Connect connected coffee machine (e.g. Bosch or Siemens).</description>
+		<semantic-equipment-tag>WhiteGood</semantic-equipment-tag>
```

Comment:
```suggestion
		<semantic-equipment-tag>CoffeeMaker</semantic-equipment-tag>
```

Reply:
Have you uploaded the new version of the xsd yet?

Reply:
Yes.